examples/custom_shape/double_wave.py

```diff
@@ -0,0 +1,49 @@
+import math
+from locust import HttpUser, TaskSet, task, constant
+from locust import LoadTestShape
+
+
+class UserTasks(TaskSet):
+    @task
+    def get_root(self):
+        self.client.get("/")
+
+
+class WebsiteUser(HttpUser):
+    wait_time = constant(0.5)
+    tasks = [UserTasks]
+
+
+class DoubleWave(LoadTestShape):
+    """
+    A shape to immitate some specific user behaviour. In this example, midday
+    and evening meal times.
+
+    Keyword arguments:
+
+        min_users -- minimum users
+        peak_one -- first peak size
+        peak_two -- second peak size
+        amplitude -- size of the test
+        time_limit -- total length of test
+    """
+    def __init__(self, min_users=20, peak_one=1, peak_two=1.5, amplitude=150, time_limit=600):
```

Comment:
This looks a little weird to me. It's not as if this constructor can be called from somewhere where these parameters are ever set?

Probably all these variables should be class level and then you can get rid of the parameters for the constructor

Reply:
True. I'll fix.